### DISCUSSION TITLE
awesome-trilium collection of third party tools for Trilium

### DISCUSSION THREAD

**Discussion by JaCoB1123** [TriliumNext/Trilium#3592] [Ideas] [answered]
Hi everyone,

I just got the idea to create an awesome-trilium project for Trilium related projects from @kooshi. I started collecting some things in my personal trilium instance. I'm not sure if creating a project makes sense. It might be easier and more visible to add a page to the wiki. What do you think? Do you know of any other projects that are missing in my list?

Thanks. 😃

**Comment by zadam** [comment]
Hi, it would be nice to have some overview of user contributed plugins, themes etc.

It can also be done on trilium wiki, I can give rights to contribute to people who would like to manage that.

**Comment by marco10x15** [comment]
Please, can be added a CSS section to the awesome-trilium to collect all the little CSS trick that are displayed on the forum.

example:
- https://github.com/zadam/trilium/discussions/3873
- https://github.com/zadam/trilium/discussions/2670

Thanks!

**Reply by meichthys** [reply]
These have been submitted to https://github.com/Nriver/awesome-trilium as a pull request.

**Comment by meichthys** [comment, marked as answer]
@JaCoB1123 There's already a good collection here: https://github.com/Nriver/awesome-trilium
